pulser/tests/test_paramseq.py

```diff
@@ -175,7 +175,8 @@ def test_str():
     sb.add(pls, "ch1")
     s = (
         f"Prelude\n-------\n{str(seq)}Stored calls\n------------\n\n"
-        + "1. add(Pulse.ConstantPulse(mul(var, 100), var, -1, var), ch1)"
+        + "1. add(Pulse.ConstantPulse(mul(var[0], 100), var[0],"
```

Comment:
I guess that it would be nice that a variable declared with `size=None` would keep being described without the `[0]`, but that's a small detail.

Reply:
Do you think we could leave it for another PR and keep the Issue open? I even wonder if the `_VariableItem` class shouldn't be removed in favor of an instanciation of `ParamObj` doing the indexation. I have some draft about it but it's having UT issues.

Reply:
Sure, that's okay with me